connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectService.scala

```diff
@@ -179,6 +182,9 @@ object SparkConnectService {
 
   private[connect] var server: Server = _
 
+  var uiTab: Option[SparkConnectServerTab] = None
+  var listener: SparkConnectServerListener = _
```

Comment:
could both be private, or `private[connect]`?

Reply:
Updated to `private[connect]`.